src/keyboardAction.js

```diff
@@ -169,8 +169,8 @@ export function dndzone(node, options) {
         switch (e.key) {
             case "Enter":
             case " ": {
-                // we don't want to affect nested input elements
-                if ((e.target.value !== undefined || e.target.isContentEditable) && !allDragTargets.has(e.target)) {
+                // we don't want to affect nested input elements or clickable elements
+                if ((e.target.click !== undefined || e.target.value !== undefined || e.target.isContentEditable) && !allDragTargets.has(e.target)) {
```

Comment:
good catch. i am not sure that checking for .click is a reliable enough test. looks like a simple div has that too.
<img width="342" alt="Screen Shot 2021-02-05 at 2 56 16 pm" src="https://user-images.githubusercontent.com/20507787/106987940-9cfbdc00-67c2-11eb-8b83-ddaa4af7e68a.png">

Reply:
Yeah, you're right about that. I meant to add a note to it in my PR saying that I wasn't sure about that heuristic. We could of course check for certain tags, like `a` or `button` but that won't work for custom elements.

It makes me think we should only activate dragging when the root drag element is selected. Intercepting keyboard input events for all nested elements and preventing them from bubbling just seems like a recipe for disaster. Maybe we should remove all specific checks and just ensure the clicked element is a drag target:

```js
if ( !allDragTargets.has(e.target)) {
```

Since drag targets are given a tabIndex, they will always be focusable, and keyboard users will be able to move stuff around as needed.

What do you think?